src/ui/server/controllers/insurance/business/turnover/index.ts

```diff
@@ -57,4 +62,59 @@ const get = (req: Request, res: Response) => {
   }
 };
 
-export { pageVariables, get };
+/**
+ * posts turnover page
+ * runs validation and either renders template with errors or redirects to next page
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.redirect} Broker page
+ */
+const post = async (req: Request, res: Response) => {
+  try {
+    const { application } = res.locals;
+
+    if (!application) {
+      return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
+    }
+
+    const { referenceNumber } = req.params;
+
+    const { body } = req;
+
+    // populate submittedValues
+    const submittedValues = {
+      [ESTIMATED_ANNUAL_TURNOVER]: body[ESTIMATED_ANNUAL_TURNOVER],
+    };
+
+    // run validation on inputs
+    const validationErrors = generateValidationErrors(body);
+
+    // if any errors then render template with errors
+    if (validationErrors && Object.keys(validationErrors).length) {
```

Comment:
I don't think we need the `Object.keys` check, the validation helper(s) will return `false` if there are no errors

Just noticed this is in "nature of your business" as well, think can be simplified